runtime/local/src/lib.rs

```diff
@@ -1467,4 +1467,70 @@ impl_runtime_apis! {
             Executive::try_execute_block(block, state_root_check, select).expect("execute-block failed")
         }
     }
+
+    #[cfg(feature = "evm-tracing")]
```

Comment:
Could you please explain why is this feature flag used?

Not sure I understand given it's runtime code.

Reply:
Technically we can build runtime without this feature, because it’s still experimental. But in near future it could be moved to default or removed.

Reply:
Removed this feature